simpleflow/execute.py

```diff
@@ -197,23 +201,37 @@ def execute(*args, **kwargs):
             with tempfile.TemporaryFile() as result_fd, tempfile.TemporaryFile() as error_fd:
                 dup_result_fd = os.dup(result_fd.fileno())  # remove FD_CLOEXEC
                 dup_error_fd = os.dup(error_fd.fileno())  # remove FD_CLOEXEC
-                # print('error_fd: {}'.format(dup_error_fd))
+                arguments_json = format_arguments_json(*args, **kwargs)
                 full_command = [
                     interpreter, '-m', command,  # execute module a script.
-                    get_name(func), format_arguments_json(*args, **kwargs),
+                    get_name(func),
                     '--logger-name={}'.format(logger_name),
                     '--result-fd={}'.format(dup_result_fd),
                     '--error-fd={}'.format(dup_error_fd),
                     '--context={}'.format(json_dumps(context)),
                 ]
+                if len(arguments_json) < MAX_ARGUMENTS_JSON_LENGTH:  # command-line limit on Linux: 128K
```

Comment:
Shouldn't we take into account the length of all the arguments (including the executable name)?

Reply:
Don't think it's needed — 64K is half the limit.
Let's discuss this again when the context is the big thing :slightly_smiling_face: